### PR TITLE
Change exec_ceph_cmd() so it won't filter anything of the command output

### PR DESCRIPTION
**Pull request by gklein**
A zero int or float which evaluates to false with the current condition, should not be filtered out.

Fixes: #1152 

Signed-off-by: Gil Klein <gklein@users.noreply.github.com>

### CODE REVIEW

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
 
         # For some commands, like "ceph fs ls", the returned output is a list
         if isinstance(out, list):
-            return [item for item in out if item]
+            return [item for item in out if not item == ""]
```
Comment (vasukulkarni) [inline ocs_ci/ocs/resources/pod.py]: I am up for dropping this line completely, let the test program handle what it needs rather than preprocess here?

Comment (gklein) [reply ocs_ci/ocs/resources/pod.py]: Fine with me, I thought there was an intention here to filter empty strings, but let remove it and run some testing to make sure it doesn't break anything.

**Comment by gklein** [comment]
> PR validation [Job UNSTABLE](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/729//testReport) (some or all tests failed).

There was only one failure in a must-gather test:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/729/testReport/tests.manage.z_cluster.test_must_gather/TestMustGather/test_must_gather/

Seems to be due to #1036

File: ocs_ci/ocs/resources/pod.py [hunk 2]
```diff
@@ -203,7 +203,7 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
 
         # For some commands, like "ceph fs ls", the returned output is a list
         if isinstance(out, list):
-            return [item for item in out if item]
+            return [item for item in out]
```
Comment (petr-balogh) [inline ocs_ci/ocs/resources/pod.py]: ok this is just copy of the out list here, this case I guess we can remove completely this line and line above and just return out on line 207 as we will return list anyway. Or do I miss something here?

Comment (petr-balogh) [reply ocs_ci/ocs/resources/pod.py]: and if we return list in some cases we should put it to returns section in the docstring. So comment in line 204 should be moved there to docstring.

File: ocs_ci/ocs/resources/pod.py [hunk 3]
```diff
@@ -188,6 +188,7 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
 
         Returns:
             dict: Ceph command output
+            (For some commands, like "ceph fs ls", the returned output is a list)
```
Comment (petr-balogh) [inline ocs_ci/ocs/resources/pod.py]: You can just start line like:

´´´
list: For some commands...
´´´

**Comment by ebenahar** [comment]
Closing. Please re-open if needed

**Pull request closed without merging**